classes/pdf/HTMLTemplateOrderSlip.php

```diff
@@ -274,10 +274,13 @@ protected function getTaxBreakdown()
     public function getProductTaxesBreakdown()
     {
         // $breakdown will be an array with tax rates as keys and at least the columns:
-        // 	- 'total_price_tax_excl'
-        // 	- 'total_amount'
+        //  - 'total_price_tax_excl'
+        //  - 'total_amount'
         $breakdown = [];
 
+        // Ignore order's discounts
+        $this->order->total_discounts_tax_excl = $this->order->total_discounts_tax_incl = 0;
```

Comment:
@shaffe-fr May be store in a temporary variable value for these variables and reassign it after `$this->order->getProductTaxesDetails`. No ?

Reply:
Yes, thanks! It's fixed.